### ISSUE TITLE
update  README reference to project purpose

### ISSUE THREAD

**Issue by apotheon** [marshallpierce/base64-serde#24]
Should the README description of the project's purpose be updated?  It currently states its existence as a stopgap solution for a pending change in another project's issue, but the issue has been closed as unimportant.

**Comment by marshallpierce** [comment]
I haven't kept track of the state of serde, so I'm not sure if this is still a good way of solving the problem. I only put this together because users wanted a not-horrible way to integrate base64 into serde... That said, it seems to be used in practice, and it does work, so 🤷

**Comment by Sytten** [comment]
I think the project should become the "official" way, I spent time searching and it seems like the current best solution

**Comment by marshallpierce** [comment]
Fair enough. Would you mind preparing a PR updating the README with what you think would have helped you more quickly assess the state of the world with serde and how this library fits in?
